### PR TITLE
Feature/tailwind update

### PR DESCRIPTION
**Pull request by andrew-boyd**
Updates the Tailwind guide to reflect the new `@formkit/tailwindcss` package. A vast improvement in my opinion.

New guide can be previewed here: https://formkit-com-git-feature-tailwind-update-formkit.vercel.app/guides/create-a-tailwind-theme

### CODE REVIEW

File: examples/guides/tailwind-theme/override/theme.js [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+    noFiles: 'block text-gray-800 text-sm mb-1',
+    fileItem: 'block flex text-gray-800 text-sm mb-1',
+    removeFiles: 'ml-auto text-blue-500 text-sm'
+  },
+  month: textClassification,
+  number: textClassification,
+  password: textClassification,
+  radio: Object.assign({}, boxClassification, {
+    input: boxClassification.input.replace('rounded-sm', 'rounded-full'),
+  }),
```
Comment (justin-schroeder) [inline examples/guides/tailwind-theme/override/theme.js:48]:
```suggestion
  radio: {
    ...boxClassification,
    input: boxClassification.input.replace('rounded-sm', 'rounded-full'),
  },
```

File: examples/guides/tailwind-theme/tailwind-theme/theme.js [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+    noFiles: 'block text-gray-800 text-sm mb-1',
+    fileItem: 'block flex text-gray-800 text-sm mb-1',
+    removeFiles: 'ml-auto text-blue-500 text-sm'
+  },
+  month: textClassification,
+  number: textClassification,
+  password: textClassification,
+  radio: Object.assign({}, boxClassification, {
+    input: boxClassification.input.replace('rounded-sm', 'rounded-full'),
+  }),
```
Comment (justin-schroeder) [inline examples/guides/tailwind-theme/tailwind-theme/theme.js:63]:
```suggestion
  radio: {
    ...boxClassification,
    input: boxClassification.input.replace('rounded-sm', 'rounded-full'),
  },
```

File: examples/guides/tailwind-theme/tailwind-theme/theme.js [hunk 3]
```diff
@@ -0,0 +1,79 @@
...
+    input: 'form-range appearance-none w-full h-2 p-0 bg-gray-200 rounded-full focus:outline-none focus:ring-0 focus:shadow-none'
+  },
+  search: textClassification,
+  select: textClassification,
+  submit: buttonClassification,
+  tel: textClassification,
+  text: textClassification,
+  textarea: Object.assign({}, textClassification, {
+    input: 'block w-full h-32 px-3 border-none text-base text-gray-700 placeholder-gray-400 focus:shadow-outline',
+  }),
```
Comment (justin-schroeder) [inline examples/guides/tailwind-theme/tailwind-theme/theme.js:75]:
```suggestion
  textarea: {
    ...textClassification,
    input: 'block w-full h-32 px-3 border-none text-base text-gray-700 placeholder-gray-400 focus:shadow-outline',
  },
```

**Review by justin-schroeder — COMMENTED** [review]
Made some suggestions

File: examples/guides/tailwind-theme/override/theme.js [hunk 4]
```diff
@@ -0,0 +1,64 @@
...
+    input: 'form-range appearance-none w-full h-2 p-0 bg-gray-200 rounded-full focus:outline-none focus:ring-0 focus:shadow-none'
+  },
+  search: textClassification,
+  select: textClassification,
+  submit: buttonClassification,
+  tel: textClassification,
+  text: textClassification,
+  textarea: Object.assign({}, textClassification, {
+    input: 'block w-full h-32 px-3 border-none text-base text-gray-700 placeholder-gray-400 focus:shadow-outline',
+  }),
```
Comment (andrew-boyd) [inline examples/guides/tailwind-theme/override/theme.js:60]:
```suggestion
  textarea: {
    ...textClassification,
    input: 'block w-full h-32 px-3 border-none text-base text-gray-700 placeholder-gray-400 focus:shadow-outline'
  },
```

**Pull request merged**